src/store/localStorage.ts

```diff
@@ -9,32 +9,31 @@ export interface LocalStorageConnectorConfig {
   properties: string[];
```

Comment:
the file is still called `localStorage`, but it is also used for `sessionStorage` now.

How about renaming it to `storage` or `localAndSessionStorage` to make it more descriptive?

Reply:
makes sense

Reply:
renamed